backend/reservations/checks.py

```diff
@@ -1,13 +1,62 @@
 from django.core.checks import Error, Warning, Info, register
 from reservations.models import Room, Guest
 
+def ticket_chain(p_guest, p_chain=[]):
+    a_chain = [p_guest] + p_chain
+
+    if not p_guest.transfer or p_guest.transfer == '':
+        return a_chain
+
+    try:
+        a_guest = Guest.objects.get(ticket=p_guest.transfer)
```

Comment:
is there any other exception type `Guest.objects.get` can return? Being this is a recursive function perhaps we should catch the top level exception(`except Exception as e`) and return `a_chain`?

Reply:
Yea if multiple `Guest` objects are returned but if that happens then there is nothing the drama checks can do for us :v